### ISSUE TITLE
[BUG] Launcher does not properly install JRE 8

### ISSUE THREAD

**Issue by iAmSpace** [PojavLauncherTeam/PojavLauncher#3571]
### Describe the bug

So, I was having an issue with Fabric whilst on the Google Play variant of the launcher, so I switched to the apk version in the GitHub Actions and it says that there is something wrong with the bundled JRE. I really do not understand so I will provide screenshots.

https://media.discordapp.net/attachments/1018272534434676827/1021905500167733328/Screenshot_20220920_230659.jpg
https://media.discordapp.net/attachments/1018272534434676827/1021905499836395540/Screenshot_20220920-230708_PojavLauncher_Minecraft_Java_Edition_for_Android.jpg

### The log file and images/videos

![Screenshot_20220920-230708_PojavLauncher (Minecraft Java Edition for Android)](https://user-images.githubusercontent.com/64274699/191373898-43b2afa7-91c1-4e16-bb98-16e1f57d270f.jpg)
![Screenshot_20220920_230659](https://user-images.githubusercontent.com/64274699/191373901-0f8c7f51-07d0-4f74-b854-93e441be3b68.jpg)

### Steps To Reproduce

```markdown
1. Install from app-debug.apk
2. Launch
3. Go to Settings or Run
```

### Expected Behavior

JRE 8 should be included... right?

### Platform

```markdown
- Device model: Samsung Galaxy S20+
- CPU architecture: aarch64
- Android version: 12
- PojavLauncher version: Latest from GitHub Actions
```

**Comment by Aerolome** [comment]
Yep, the latest one is broken. Just use the secons latest from a month ago.
